pkg/kubelet/kubelet.go

```diff
@@ -110,6 +111,18 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	etcHostsPath = "/etc/hosts"
+
+	// Capacity of the channel for recieving pod lifecycle events. This number
+	// is a bit arbitrary and may be adjusted in the future.
```

Comment:
Yes, it could be. I don't think there is much value of calibrating this now, but we can certainly do that later.